src/components/AddItemForm/AddItemForm.js

```diff
@@ -140,7 +140,7 @@ const AddItemForm = ({ listId }) => {
         />
         <label
           htmlFor="notSoonOption"
-          className="add-item-form__label add-item-form__label_type_radio label"
+          className="add-item-form__label add-item-form__label_type_radio label label_check-radio"
```

Comment:
Is the additional class just for the future?

Reply:
Yes, we added this because we thought we'd want a generic class for labels for checkbox/radio buttons, since we will almost certainly style them similarly (as in beside the check/radio, instead of on a separate line).